middlewares/verify_token.js

```diff
@@ -0,0 +1,27 @@
+const jwt = require("jsonwebtoken");
+const auth_config = require("../config/auth.config");
+require("dotenv").config();
+
+const verify_token = async (req, res, next) => {
+  const req_token = req.get("Authorization");
```

Comment:
Along with token,  we can also send/receive user id. It is generally done in organisations.